### PR TITLE
Add new rares, 11 at last count

### PR DESCRIPTION
**Pull request by libraryaddict**
One previously unknown, other hoarded

### CODE REVIEW

File: mafia/data/raretiles.json [hunk 1]
```diff
@@ -220,6 +224,8 @@
   { "minute": 2566, "row": 10, "column": 0 },
   { "minute": 2568, "row": 8, "column": 3 },
   { "minute": 2577, "row": 4, "column": 4 },
+  { "minute": 2592, "row": 10, "column": 3 },
+  { "minute": 2605, "row": 7, "column": 5 },
```
Comment (Veracity0) [inline mafia/data/raretiles.json:228]: You found one of the 34 new rares that I have discovered.
The other 9 you have found are new to me.

I'm going to commit my discoveries to BeachComber, soon.
After we've been through tides=0 again, in 2 days.

For now, I have added your discoveries to my "known" rare lists.

Comment (libraryaddict) [reply mafia/data/raretiles.json:228]: Yeah that's fine, I will note that I too plan to hit the beaches on tides 0, which is probably going to be 7.5 to 10k

May the fastest win.
Though I will also note that my script will probably hit them only once each, so if there's more than one sparkle..
image

Comment (libraryaddict) [reply mafia/data/raretiles.json:228]: Well, I found nothing new

File: mafia/data/raretiles.json [hunk 2]
```diff
@@ -538,6 +548,7 @@
   { "minute": 6350, "row": 6, "column": 2 },
   { "minute": 6376, "row": 7, "column": 6 },
   { "minute": 6380, "row": 6, "column": 1 },
+  { "minute": 6301, "row": 8, "column": 8 },
```
Comment (Veracity0) [inline mafia/data/raretiles.json]: This is between 6380 and 6391.
You call it 6301.
Did you mean 6381?

**Review by Veracity0 — CHANGES_REQUESTED** [review]

File: mafia/data/raretiles.json (hunk 2 above)
Comment (libraryaddict) [reply mafia/data/raretiles.json]: Its accurate but wrong location, I just copy/paste it from what the bot reports.

**Pull request merged**